tests/test_stitcher.py

```diff
@@ -190,13 +190,27 @@ def test_download_dem(dem_name):
         # Within the Los Angeles tile
         bounds = [-118.8, 34.6, -118.5, 34.8]
 
-    dem_arr, _ = stitch_dem(bounds,
+    dem_arr, p = stitch_dem(bounds,
                             dem_name,
                             n_threads_downloading=5,
                             dst_ellipsoidal_height=True,
                             dst_resolution=0.0002777777777777777775
                             )
     assert len(dem_arr.shape) == 2
+    assert np.isnan(p['nodata'])
+
+
+def test_boundary_of_missing_glo_30_data():
+    # See https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/issues/89#issuecomment-1399142499
+    bounds = [42.0, 37.0, 44.0, 39.0]
+    dem_arr, p = stitch_dem(bounds,
+                            'glo_30',
+                            n_threads_downloading=5,
+                            dst_ellipsoidal_height=True,
+                            dst_resolution=0.0002777777777777777775
+                            )
+    assert len(dem_arr.shape) == 2
+    assert np.isnan(p['nodata'])
```

Comment:
This is the full integration test for the bug caught by @forrestwilliams - thank you!

See https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/issues/89#issuecomment-1399142499